languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -14957,6 +14957,35 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
             <message>Das Adverb <suggestion>immerhin</suggestion> wird zusammengeschrieben.</message>
             <example correction="Immerhin"><marker>Immer hin</marker> war der Laden sehr günstig, wenn das Essen schon mittelmäßig war.</example>
         </rule>
+        <rule id="IMMER_HIN2" name="immer hin (immerhin)" default="temp_off"> <!-- TEST -->
+            <antipattern>
+                <token>immer</token>
+                <token>hin</token>
+                <token>und</token>
+                <token>her</token>
+                <example>Der Linienbus fährt immer hin und her.</example>
+            </antipattern>
+            <antipattern>
+                <token>immer</token>
+                <token>hin-</token>
+                <token>und</token>
+                <token regexp="yes">her*</token>
+                <example>Sie war immer hin- und hergerissen.</example>
+            </antipattern>
+            <antipattern>
+                <token>immer</token>
+                <token>hin</token>
+                <token>und</token>
+                <token>wieder</token>
+                <example>Ich denke noch immer hin und wieder an das Buch.</example>
+            </antipattern>
+            <pattern>
+                <token>immer</token>
+                <token>hin</token>
+            </pattern>
+            <message>Das Adverb <suggestion><match no="1" /><match no="2" /></suggestion> wird zusammengeschrieben.</message>
```

Comment:
Das zweite token in der suggestion könntest du evtl. 'alllower' setzen :-)

Reply:
Danke, hab's noch ergänzt. Allerdings steht unten jetzt "This branch has conflicts that must be resolved"?